### ISSUE TITLE
Images Missing In Action

### ISSUE THREAD

**Issue by MWE001** [IntegraMOD/IntegraMOD151#59]
Somewhere along the line we lost an entire folder of images. /images/block_images contains all the images that live in the upper right corner of the blocks. That folder is absent from this very version that is hosted on Git. Is this a folder that can simply be nixed off of the Integra site and dropped in and called fixed and closed?

**Comment by MWE001** [comment]
Or ..... Is that style select block a drop in from the current Integramod website and all the other blocks are imageless to begin with? I am kinda dazed and confused here. It's been so long It's like a momentary lapse of reason.

**Comment by MWE001** [comment]
http://prntscr.com/lovvq7

**Comment by MWE001** [comment]
Ill just add this one here as a missing image as well instead of opening another issue. No point in it.  

This one is Private message Bar. Without an image this looks a shade rough to say the least.

![screenshot_4](https://user-images.githubusercontent.com/2566493/49282352-95a74e00-f454-11e8-8586-a47f819a2bbf.jpg)

**Comment by MWE001** [comment]
Hey these images missing is NOT an issue after all. I realized it this morning that is was being converted to Font Awesome but I was too tired to update before bed. John and I posted about it on Integras forum. We can close this one here and I will update the post over there as needs be when I find missing stuff or I can post here on a closed issue. The choice is up to you fellas. Either way is all good by me. :-) If I try to remember where I saw something missing, well, I will forget to tell you how that one will turn out. ;-)
